certora/specs/RevertsERC20.spec

```diff
@@ -8,40 +8,31 @@ methods {
     function _._moveDelegateVotes(address, address, uint256) internal => CONSTANT;
 }
 
-// Check that transfers with zero address or too large amounts revert.
-rule transferReverts(env e, address to,uint256 amount) {
-    // Safe require as transfer is a non-payable function.
-    require e.msg.value == 0;
-
+// Check the revert conditions for the transfer function.
+rule transferRevertConditions(env e, address to,uint256 amount) {
```

Comment:
```suggestion
rule transferRevertConditions(env e, address to, uint256 amount) {
```

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/98/commits/9025f3b5c34e16082305e6d4b97dc5bf96eb67f5.